docs/maya/glue.md

```diff
@@ -0,0 +1,145 @@
+# AdnGlue
+
+AdnGlue is a Maya node that can be used to attach multiple muscles together. This solver uses as input a series of simulated muscles and allows the user to define glue connections between those muscles by using a maximum distance value and painted maps.
+
+The maps required to control the AdnGlue solver can be painted using the standard Maya Paint tool.
```

Comment:
Would you add this to a designated note?

Reply:
If you agree, I would do that in the Simple Setup page (as we do for other deformers)

Reply:
Agreed!